src/assets/scss/input-box.scss

```diff
@@ -0,0 +1,43 @@
+.text-input-box {
+    width: 600px;
```

Comment:
You should not define a fixed length for the input. The input must grow or shrink based on the container that is put in.

You should define a min-width for the input.

Reply:
Done! added a minimum width of 200px and the size configured according to the container